piton/tools/src/sims/sims,2.0

```diff
@@ -2221,7 +2221,7 @@ sub assemble_diag
     }
 
     if ($opt{blackparrot} and $opt{precompiled}) {
-      push @{$opt{asm_diag_root}}, "$ENV{HOME}/scratch/pre-alpha-release/bp_common/test/mem/";
+      push @{$opt{asm_diag_root}}, "$ENV{HOME}/BlackParrot/black-parrot/bp_common/test/mem/";
```

Comment:
Can we change this to point inside the bp submodule?

Reply:
By which I mean use `$dv_root/piton/design/chip/tile/blackparrot/bp_common/test/mem` or whatever

Reply:
Well, I didn't update the tests submodule yet in the blackparrot submodule. So was pointing to my own repo's tests. 
But it can be done.

Reply:
So basically `$ENV{DV_ROOT}/piton/design/chip/tile/blackparrot/bp_common/test/mem`?

Reply:
I forget whether sims,2.0 should specifically use `$ENV{DV_ROOT}` or `$dv_root` because I've seen both. I think what you said is fine though. I doubt it's overwriting `$ENV{DV_ROOT}`